docs/spec/index.md

```diff
@@ -1,54 +1,104 @@
+### Structure
 
-## Field Definitions
+There are two types of files used in the TODS standard:
 
-### deadheads.txt
+- **Supplement files**, used to add, modify, and delete information from public GTFS files to model the operational service for internal purposes (with a `_supplement` filename suffix).
+- **TODS-Specific files**, used to model operational elements not currently defined in the GTFS standard.
 
-| **Field Name** | **Type** | **Required** | **Description** |
-| --- | --- | --- | --- |
-| deadhead_id | ID | Required | Identifies a deadhead. |
-| service_id | ID referencing [**calendar.service_id**](https://developers.google.com/transit/gtfs/reference#calendartxt) | Required | Identifies a set of dates when the deadhead is scheduled to take place. |
-| block_id | ID | Required | Identifies the block to which the deadhead belongs. |
-| shape_id | ID referencing [**shapes.shape_id**](https://developers.google.com/transit/gtfs/reference#shapestxt) | Optional | Identifies a geospatial shape that describes the vehicle travel path for a deadhead. |
-| to_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately following to the deadhead within the block_id. |
-| from_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately prior to the deadhead within the block_id. |
-| to_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately following the deadhead within the block_id. |
-| from_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately prior to the deadhead within the block_id. |
+### Files
 
-### ops_locations.txt
+| **File Name** | **Type** | **Description** |
+| --- | --- | --- |
+| trips_supplement.txt | Supplement | Supplements and modifies GTFS [trips.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#tripstxt) with deadheads and other non-public trip information. |
+| stops_supplement.txt | Supplement | Supplements and modifies GTFS [stops.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) with internal stop locations, waypoints, and other non-public stop information.|
+| stop_times_supplement.txt | Supplement | Supplements and modifies GTFS [stop_times.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt) with non-public times at which trips stop at locations, and related information. |
+| routes_supplement.txt | Supplement | Supplements and modifies GTFS [routes.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#routestxt) with internal route identifiers and other non-public route identification. |
```

Comment:
I'm working on updating the examples to the new spec, and something I ran into: What route are deadheads on?

It's a little awkward cuz routes don't describe a path a vehicle takes. Instead, they describe a public-facing name that a trip has. But deadheads aren't public facing, so they don't have an identity like that.

Should the deadhead's route be the route that it's pulling out to? Some global "deadhead" route? A different route for every different type of deadhead? Can we make the `trips_supplement.txt/route_id` field optional?

Reply:
I don't think this is something that we should define, as most existing agencies already have a standard practice per their scheduling system that they can continue to use.

For example, some systems (a) assign the trips to the route of their adjacent revenue trip, some (b) assign it to a "deadhead" route, some (c) assign it to a specific deadhead route by garage; all of those approaches are supported in the specification provided the routes are defined in routes_supplement.

Likewise, I think the more that we stray from the GTFS structure (i.e. saying things in static GTFS are not required), the more opportunity there is for confusion or breaking of existing architecture, so I would not recommend making it optional.

I think a fairly universal principle we should uphold is that the set of Supplemented GTFS files (static GTFS schedule files after having applied the Supplement modifications) should still validate in accordance in accordance with the canonical GTFS validator.

Perhaps it's worth:
- Including the above definition of Supplemented GTFS (Static GTFS after TODS Supplement modifications), and the notion that it should validate, in the documentation?
- Adding a "best practices" section akin to the equivalent in static GTFS, or perhaps even something akin to a "statement of legislative intent" that records these decisions being made with intentionality?